hoomd/md/pytest/test_methods.py

```diff
@@ -0,0 +1,23 @@
+import hoomd
+import pytest
+import numpy
+import itertools
+from copy import deepcopy
+
+_int_methods = [
+    hoomd.md.methods.Langevin(hoomd.filter.All(), kT=1, seed=1),
+    hoomd.md.methods.NVT(hoomd.filter.All(), kT=1, tau=1),
+]
+
+@pytest.fixture(scope='function', params=_int_methods)
```

Comment:
Use the `ids` parameter so the tests get a useful name in the output. Without the ids, the tests get named like this:
```
hoomd/md/pytest/test_methods.py::test_attach[CPU-int_method0] PASSED                                                    [ 25%]
hoomd/md/pytest/test_methods.py::test_attach[CPU-int_method1] PASSED                                                    [ 50%]
```
When it could say `CPU-Langevin` and `CPU-NVT`.